config/locales/es/management.yml

```
@@ -131,3 +131,4 @@ es:
       create:
         success: Se han enviado <strong>%{count} invitaciones</strong>.
         title: Enviar invitaciones
+    need_managed_user: Para realizar esta acción debe seleccionar un usuario.
```

```suggestion
    need_managed_user: Para realizar esta acción debes seleccionar un usuario.
```

I think in other places in this file we use "tú" to address the user.